javaagent/javaagent.gradle

```
@@ -42,8 +42,14 @@ CopySpec isolateSpec(Collection<Project> projectsWithShadowJar) {
 shadowJar {
   dependsOn ':instrumentation:shadowJar'
   dependsOn ':javaagent-exporters:shadowJar'
-  def projectsWithShadowJar = [project(':instrumentation'), project(':javaagent-exporters')]
+  def projectsWithShadowJar = [project(':javaagent-exporters'), project(':instrumentation')]
```

better to prefer classes that we've used to build/test instrumentation?
```suggestion
  def projectsWithShadowJar = [project(':instrumentation'), project(':javaagent-exporters')]
```

---

Yeah that's what I mentioned in the description about reversing order. This order doesn't work, no clue why :( I filed #1135 to follow up since there does seem to be something amiss but would be nice to unblock my aws distro 😅